src/repositories/projectsRepository.ts

```diff
@@ -0,0 +1,23 @@
+import { Db } from 'mongodb';
+import { IConfig } from 'config';
+import { BaseRepository } from './baseRepository';
+import { ILogger } from '../services/logger';
+
+/**
+ * Project information from docs_metadata.projects for parser builds.
+ */
+export class ProjectsRepository extends BaseRepository {
+  constructor(db: Db, config: IConfig, logger: ILogger) {
+    super(config, logger, 'ProjectsRepository', db.collection('projects'));
```

Comment:
Like @branberry mentioned above with `docs_metadata`, could we make `projects` a constant like in the other ____Repository classes

Reply:
@branberry @mmeigs to confirm, you mean like with the example of the docsets collection where it's set in `worker-env-construct.ts`from aws parameter store as shown [here](https://github.com/mongodb/docs-worker-pool/blob/025a8722b5aa69da322b4fbbc6cdee05f3d510e7/cdk-infra/lib/constructs/worker/worker-env-construct.ts#L67)?

Reply:
I added `projects` as a constant as done with the collections for the other repository classes except that I didn't configure it to pull the value from AWS parameter store. If that looks good to you both I'll do the same with `docs_metadata` as a constant